runtime/bcverify/staticverify.c

```diff
@@ -1767,6 +1767,24 @@ j9bcv_verifyClassStructure (J9PortLibrary * portLib, J9CfrClassFile * classfile,
 			goto _formatError;
 		}
 
+		/* The requirement for taking no arguments was introduced in Java SE 9.
+		 * In a class file whose version number is 51.0 or above, the method
+		 * has its ACC_STATIC flag set and takes no arguments.
+		 */
+                 /* Leave this here to find usages of the following check:
+                  * if (J2SE_VERSION(vm) >= J2SE_19) {
+                  */
+		if (vmVersionShifted >= BCT_Java9MajorVersionShifted) {
+			if (classfile->majorVersion >= 51) {
+				if ((CFR_METHOD_NAME_CLINIT == isInit) 
+					&& (0 != argCount) 
+				) {
```

Comment:
Nitpick: This condition is reasonably short - it could get away with being a single line 
```
if ((CFR_METHOD_NAME_CLINIT == isInit) && (0 != argCount)) {
```

Reply:
Given how long this has waited for me to finish the review (sorry) I'm going to merge it as is.  If you want to address this nitpick, feel free to open a new PR (or not).

Reply:
or more precisely, I'll kick off the PR build and merge once the build succeeds.